### PR TITLE
SqlServer integration testing

### PR DESCRIPTION
**Pull request by michaelvidal24**
This change is [Reviewable](https://reviewable.io/reviews/healthcatalyst/fabric.identity/103)

### CODE REVIEW

File: Fabric.Identity.API/Persistence/SqlServer/Scripts/create_identity_database.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿DECLARE @dbname nvarchar(128)
```
Comment (bsmithvt) [inline Fabric.Identity.API/Persistence/SqlServer/Scripts/create_identity_database.sql]: Do we still need this SQL script?

File: Fabric.Identity.API/Persistence/SqlServer/Scripts/create_identity_tables.sql [hunk 2]
```diff
@@ -1,3 +1,5 @@
+USE [Identity]
```
Comment (bsmithvt) [inline Fabric.Identity.API/Persistence/SqlServer/Scripts/create_identity_tables.sql]: Do we still need this SQL script?

File: Fabric.Identity.IntegrationTests/IntegrationTestsFixture.cs [hunk 3]
```diff
@@ -61,12 +77,44 @@ public IntegrationTestsFixture(string storageProvider = FabricIdentityConstants.
 
         private static ICouchDbSettings CouchDbSettings => _settings ?? (_settings = new CouchDbSettings
         {
-            DatabaseName = "integration-" + DateTime.UtcNow.Ticks,
-            Username = "",
-            Password = "",
+            DatabaseName = $"integration-{DatabaseNameSuffix}",
```
Comment (bsmithvt) [inline Fabric.Identity.IntegrationTests/IntegrationTestsFixture.cs]: Should we remove these credentials?

File: Fabric.Identity.IntegrationTests/IntegrationTestsFixture.cs [hunk 4]
```diff
@@ -229,6 +306,89 @@ private static IS4.ApiResource GetTestApiResource()
...
+            using (var identityContext = IdentityDbContext)
+            {
+                identityContext.Database.ExecuteSqlCommand(DeleteDataSql);
+                identityContext.ApiResources.Add(apiResource.ToEntity());
+                identityContext.Clients.Add(client.ToEntity());
+                identityContext.SaveChanges();
+            }
+        }
+
+        private static readonly string DeleteDataSql =
```
Comment (kpaul) [inline Fabric.Identity.IntegrationTests/IntegrationTestsFixture.cs]: i dont think we need to run this anymore if we are creating a new DB for each test run

File: Fabric.Identity.API/Persistence/SqlServer/Scripts/create_identity_database.sql (hunk 1 above)
Comment (michaelvidal24) [reply Fabric.Identity.API/Persistence/SqlServer/Scripts/create_identity_database.sql]: @bsmithvt: What were we using it for originally? The integration tests use the sql generated from the dacpac to create the database, and we'll use the dacpac to deploy the db everywhere else.

File: Fabric.Identity.API/Persistence/SqlServer/Scripts/create_identity_tables.sql (hunk 2 above)
Comment (michaelvidal24) [reply Fabric.Identity.API/Persistence/SqlServer/Scripts/create_identity_tables.sql]: @kpaul - can you clarify if this script is needed?

Comment (kpaul) [reply Fabric.Identity.API/Persistence/SqlServer/Scripts/create_identity_tables.sql]: i dont think we need it anymore as all the db entities are created via the dacpac

**Pull request merged**